content/_includes/site/hire-us.njk

```diff
@@ -0,0 +1,25 @@
+{% import 'embed.macros.njk' as embed %}
+
+{%- set title = '<span class="title-action">Hire Us to Build</span> Custom Web Apps <span class="title-description">That stand the test of time</span>' -%}
+
+<aside data-layout="cta" aria-labelledby="hire-cta-title" class="hire-us-cta" data-ccs-colors="dark">
+  {% set cta_content %}
+    <h2 id="hire-cta-title" data-header='stacked hire-cta'>{{ title | mdInline | safe }}</h2>
+
+    <a href='/contact/' data-btn="link-button" role="button" class="section-link" draggable="false"> Let's chat </a>
+    {{ contents | md | safe }}
+  {% endset %}
+
+  {{ embed.media_block(
+      name='hire-cta',
+      media=embed.screen(
+        fill='#78080A',
+        content=embed.img(
+          src='work/adobe/vintage-mobile.jpg',
+          alt='Mobile view of the vintage font tag question',
+          sizes='media'
+        )
+      ),
+      content=cta_content
```

Comment:
If we use `{% call embed.media_block(...) %}` here, we could put `cta_content` inline instead of in a separate variable. Not a big deal, just might be more readable.

Reply:
fixed in https://github.com/oddbird/oddleventy/pull/808/commits/b0426139237a6137623da2ce46314a44fe0de654